indico/modules/bootstrap/controllers.py

```diff
@@ -55,14 +56,14 @@ def _handle_submit(self, setup_form):
         user.last_name = setup_form.last_name.data
         user.affiliation = setup_form.affiliation.data
         user.email = setup_form.email.data
-        user.is_admin = True
 
         identifier = setup_form.username.data if config.LOCAL_USERNAMES else str(uuid4())
         identity = Identity(provider='indico', identifier=identifier, password=setup_form.password.data)
         user.identities.add(identity)
 
         db.session.add(user)
         db.session.flush()
+        grant_admin(user)
```

Comment:
Not sure about this one, it's part of the initial bootstrapping process, not something you can do later.